### PR TITLE
OU-259: Migrate Dev > Observe > Dashboard Page 

### PR DESCRIPTION
**Pull request by zhuje**
## Overview 
This PR aims to consolidate code for the Developer perspective > Observe > Dashboard page. We will remove the code that renders this page from openshift/console. The page will now be rendered by openshift/monitoring-plugin through this PR: https://github.com/openshift/monitoring-plugin/pull/167. 

## Related CONSOLE Jira Issue: 
https://issues.redhat.com/browse/CONSOLE-4236

## Related Observability UI Jira Issue: 
https://issues.redhat.com/browse/OU-259

## Testing
Must be tested with PR: https://github.com/openshift/console/pull/14192

This PR openshift/console#14192 removes the Developer perspective > Observe > Dashboard page
This PR openshift/monitoring-plugin#167 adds the Developer perspective > Observe > Dashboard page 

**Expected Results:** 
All behaviors should be the same as before the migration. 

image

**Figure 1**. Shows the Developer Perspective > Observe > Dashboard Page

### CODE REVIEW

**Comment by zhuje** [comment]
/hold 

(pending approval of https://github.com/openshift/console/pull/14192)

File: web/src/components/alerting.tsx
Comment (PeterYurkovich) [inline web/src/components/alerting.tsx:1121]: I don't believe that the dashboards page (or the metrics page above it) need to be in the `PollerPages` component. They don't need to fetch the alerts and rules info, so they can be moved back down to the `MonitoringUI` component.

**Review by PeterYurkovich — CHANGES_REQUESTED** [review]

File: web/src/components/alerting.tsx
Comment (zhuje) [reply web/src/components/alerting.tsx]: moved both Metrics and Dashboard Routes into <MonitoringUI /> in the latest commit

**Pull request merged**
